### ISSUE TITLE
saber-node.js undefined error

### ISSUE THREAD

**Issue by iCyris** [egoist/saber#40]
Hello @egoist ,

I have published [saber-theme-ume](https://www.npmjs.com/package/saber-theme-ume) on `npmjs.com`, but I met an error when run `npm run dev` (It works well when I run `npm run  dev:example` on the [theme](https://github.com/iCyris/Ume) folder):

```
$ npm run dev

> @ dev /Users/cyris/Desktop/umeeee/my-blog-3
> saber

info Using theme: /Users/cyris/Desktop/umeeee/my-blog-3/node_modules/saber-theme-ume/dist
info afterPlugins (theme-node-api)
info afterPlugins (user-node-api)
info beforeRun (builtin:source-pages)
info initPages (theme-node-api)
info initPages (user-node-api)
info onCreatePages (query-posts)
info onCreatePages (theme-node-api)
error TypeError: Cannot read property 'type' of undefined
    at /Users/cyris/Desktop/umeeee/my-blog-3/node_modules/saber-theme-ume/dist/saber-node.js:4:28
    at Array.filter (<anonymous>)
    at Saber.exports.onCreatePages (/Users/cyris/Desktop/umeeee/my-blog-3/node_modules/saber-theme-ume/dist/saber-node.js:3:46)
    at tapInfo.fn.args (/Users/cyris/Desktop/umeeee/my-blog-3/node_modules/saber/lib/index.js:67:20)
    at Promise (eval at create (/Users/cyris/Desktop/umeeee/my-blog-3/node_modules/tapable/lib/HookCodeFactory.js:65:10), <anonymous>:25:1)
    at new Promise (<anonymous>)
    at AsyncSeriesHook.eval [as promise] (eval at create (/Users/cyris/Desktop/umeeee/my-blog-3/node_modules/tapable/lib/HookCodeFactory.js:65:10), <anonymous>:4:8)
    at AsyncSeriesHook.lazyCompileHook (/Users/cyris/Desktop/umeeee/my-blog-3/node_modules/tapable/lib/Hook.js:154:20)
    at api.hooks.beforeRun.tapPromise (/Users/cyris/Desktop/umeeee/my-blog-3/node_modules/saber/lib/plugins/source-pages.js:99:35)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ dev: `saber`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Is there something wrong with the [saber-node.js](https://github.com/iCyris/Ume/blob/master/src/saber-node.js) file or I just I missed some steps when running it?

**Comment by egoist** [comment]
oops it's transpilation error from vue-compile:
<img width="1000" alt="屏幕快照 2019-03-30 上午11 49 11" src="https://user-images.githubusercontent.com/8784712/55271049-e4b99e80-52e1-11e9-8c08-506f0f1f9f2a.png">

`this.pages.values()` is a `Map` so it shouldn't use `.concat`

**Comment by egoist** [comment]
Should be fixed by `vue-compile@0.5.3`

**Comment by iCyris** [comment]
Fixed. thank u :D
